src/mscorlib/shared/System/ReadOnlySpan.cs

```diff
@@ -57,7 +57,7 @@ public ReadOnlySpan(T[] array, int start)
             if ((uint)start > (uint)array.Length)
                 ThrowHelper.ThrowArgumentOutOfRangeException();
 
-            _pointer = new ByReference<T>(ref Unsafe.Add(ref JitHelpers.GetArrayData(array), start));
+            _pointer = new ByReference<T>(ref Unsafe.Add(ref Unsafe.As<byte, T>(ref array.GetRawSzArrayData()), start));
```

Comment:
Does having to pass `ref array.GetRawSzArrayData()` to `Unsafe.As<byte, T>` incur significant performance cost?